tests/examples/test_examples.py

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#pylint: disable=no-member, too-many-locals, too-many-branches, no-self-use, broad-except, lost-exception, too-many-nested-blocks, too-few-public-methods, invalid-name
+"""
+    This file tests and ensures that the examples run out of the box.
+
+"""
+import os
+import sys
+import subprocess
+import logging
+
+def _run_command(test_name, command):
+    """Runs the script using command
+
+    Parameters
+    ----------
+    command : list of str
+        the command that needs to be run in form of list
+        Example : ['ls', '-l']
+
+    Returns
+    -------
+        True if there are no warnings or errors.
+    """
+    errors = []
+    logging.info("Running test for {}".format(test_name))
+    try:
+    	subprocess.check_call(command)
+    except Exception as err:
+        err_msg = str(err)
+        errors.append(err_msg)
+        if errors:
+            logging.error('\n'.join(errors))
+            return False
+    return True
+
+def test_cifar_default():
+    example_dir = os.path.join(os.path.dirname(__file__), '..', '..', 'examples','image_classification')
+    temp_dir = 'tmpdir'
```

Comment:
https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory

Reply:
Are the paths in the examples hardcoded or what's the issue here? Could you point me to some links?

Reply:
The path inside example scripts are hardcoded. 
https://github.com/apache/incubator-mxnet/blob/master/example/image-classification/train_cifar10.py#L26